aspnet/getting-started/installing-on-linux.rst

```diff
@@ -100,10 +100,63 @@ You should now be able to restore packages from both the official public feed on
 
 
 Installing on CentOS, Fedora and derivatives
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------------------------
 
-.. note::
+ The follwoing instrcution were tested using CentOS 7.
+ 
+Install Mono
```

Comment:
I think you make a good point here.  What would you think about noting that it is optional if you intend to just use CoreCLR?